types/relay-runtime/lib/util/isPromise.d.ts

```diff
@@ -0,0 +1 @@
+export function isPromise(p: any): p is Promise<any>;
```

Comment:
I just noticed that [the upstream export](https://github.com/facebook/relay/blob/9ef158d08bc6d1f98c352aa6e1b610807e2c779f/packages/relay-runtime/util/isPromise.js#L21) is a `default` [commonjs] export, so this should probably be:

```suggestion
export default function isPromise(p: any): p is Promise<any>;
```

Reply:
Why use the default?
I aligned myself on how this other function was handled
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/c7eecc1bb03bc15dbab434ec7cc73cc6ddf81781/types/relay-runtime/lib/util/deepFreeze.d.ts#L1

Reply:
```ts
export function isPromise(p: any): p is Promise<any>;
```

Is the equivalent of the following CommonJS export:

```js
module.exports = { isPromise };
```

…which is not how the function is actually exported in `relay-runtime/lib/util/isPromise.js`. With your definition, a person importing the function directly would get a runtime error:

```ts
import { isPromise } from "relay-runtime/lib/util/isPromise"

isPromise(…) // will throw something like `undefined is not a function`
```

Reply:
you are right :) https://github.com/DefinitelyTyped/DefinitelyTyped/pull/45515/commits/7878a2053cc659a2ee3491fbd626afc0e1273c17

So deepFreeze should also be fixed 
https://github.com/facebook/relay/blob/9ef158d08bc6d1f98c352aa6e1b610807e2c779f/packages/relay-runtime/util/deepFreeze.js#L36

Reply:
Hah, yep indeed. Mind adding that too?

Reply:
done :) 
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/45515/commits/552400a578e0ab808885afeb766212f28b737c6c

Reply:
Great, thanks!